schema/ga4gh.yaml

```diff
@@ -23,7 +23,7 @@ identifiers:
     Allele: VA
     VariationSet: VS
     Text: VT
-    # Genotype: VG
+    Genotype: GT
     Haplotype: VH
```

Comment:
since we are diverging from the Vx form of prefixes for Genotype, we may want to change Haplotype to HT? Again, we can put this under the heading of "improvement to draft data types". It would provide some consistency and justify the "GT" prefix.  Or maybe we simply call all of this "Trial Use" and fix it in 2.0?